google/cloud/pubsub/internal/subscription_session.cc

```diff
@@ -203,16 +203,9 @@ future<Status> CreateTestingSubscriptionSession(
       "test-client-id", options, std::move(retry_policy),
       std::move(backoff_policy));
 
-  auto cq = executor;  // need a copy to make it mutable
-  auto timer = [cq](std::chrono::system_clock::time_point) mutable {
-    return cq.MakeRelativeTimer(std::chrono::milliseconds(50))
-        .then([](future<StatusOr<std::chrono::system_clock::time_point>> f) {
-          return f.get().status();
-        });
-  };
-  auto lease_management = SubscriptionLeaseManagement::CreateForTesting(
-      executor, shutdown_manager, timer, std::move(batch),
-      options.max_deadline_time(), options.max_deadline_extension());
+  auto lease_management = SubscriptionLeaseManagement::Create(
```

Comment:
I wonder if the enclosing function (`CreateTestingSubscriptionSession()`) is even needed now?

Reply:
I think you are right. I have moved the method, which exists only for testing, into the test file.

PTAL

Reply:
I think I messed up my `std::move()`'s